### PR TITLE
PMM-4052 Fix order of fields in Swagger.

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-443/

**Comment by BupycHuk** [comment]
@AlekSi PTAL

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-443/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-443/

File: api/Makefile [hunk 1]
```diff
@@ -19,6 +19,7 @@ gen: clean            ## Generate files.
 					../vendor/github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway \
 					../vendor/github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger \
 					../vendor/github.com/go-swagger/go-swagger/cmd/swagger \
+					../vendor/github.com/bupychuk/swagger-order \
```
Comment (AlekSi) [inline api/Makefile]: The same case problem.

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-443/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-443/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-443/

File: Gopkg.toml [hunk 2]
```diff
@@ -44,7 +45,8 @@ required = [
   version = "=1.11.1"
 [[constraint]]
   name = "github.com/go-swagger/go-swagger"
-  version = "=0.20.1"
+  source = "github.com/Percona-Lab/go-swagger"
+  version = "=0.20.1-percona2"
```
Comment (AlekSi) [inline Gopkg.toml]: Let's add a short comment and a link to PR explaining when we can switch to the upstream.

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-443/

**Pull request merged**